### DISCUSSION TITLE
Autocomplete not displaying error when touched

### DISCUSSION THREAD

**Discussion by ErikParso** [lookfirst/mui-rff#1076] [Q&A]
**Describe the bug**
As other fields, if autocomplete value is invalid and user touches the field, it should show error.

**To Reproduce**
1. add validation to autocomplete (to be mandatory f.e.)
2. Focus autocomplete
3. Blur autocomplete
4. Autocomplete not showing error

**Expected behavior**
Autocomplete should be red and show error message

**Comment by lookfirst** [comment]
Hi Erik, this probably needs a code example.

**Comment by ErikParso** [comment]
fixed here https://github.com/lookfirst/mui-rff/issues/739
